samtranslator/internal/managed_policies.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+from pathlib import Path
+from typing import Dict, Optional
+
+from samtranslator.internal.types import GetManagedPolicyMap
+from samtranslator.translator.arn_generator import ArnGenerator
+
+with Path(os.path.dirname(os.path.abspath(__file__)), "data", "managed_policies.json").open(encoding="utf-8") as f:
+    _BUNDLED_MANAGED_POLICIES: Dict[str, Dict[str, str]] = json.load(f)
+
+
+def _dict_get(d: Optional[Dict[str, str]], k: str) -> Optional[str]:
+    if isinstance(d, dict) and k in d:
```

Comment:
adding a check like this suppress the error what `d` is not a dict, which should be detected before sam-t is shipped.